From ad28c0fc2f6e8007325cc643c98665c7cb113cb3 Mon Sep 17 00:00:00 2001
From: unknown <pair13@.ISLLCNETS.Studentloan.org>
Date: Wed, 13 Aug 2008 17:16:12 -0500
Subject: [PATCH] Added speedbar hotkey

---
 mine/customizations.el |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/mine/customizations.el b/mine/customizations.el
index 4245732..a9a8ca9 100644
--- a/mine/customizations.el
+++ b/mine/customizations.el
@@ -2,6 +2,7 @@
 (global-set-key "\C-x\C-m" 'execute-extended-command)
 (global-set-key "\C-x\C-k" 'kill-region)
 (global-set-key "\C-x\C-k" 'kill-region)
+(global-set-key "\C-x\C-j" 'speedbar)
 (global-unset-key "\C-z")
 
 ;; Setup Environmental Variables
-- 
1.5.6.1.1071.g76fb

